engine/src/main/java/org/destinationsol/assets/sound/SpecialSounds.java

```diff
@@ -106,4 +107,48 @@ public void playColl(SolGame game, float absImpulse, SolObject o, Vector2 positi
         }
         game.getSoundManager().play(game, metal ? metalColl : rockColl, position, o, absImpulse * Const.IMPULSE_TO_COLL_VOL);
     }
+
+    /**
+     * Gets the damage sound associated with the given {@link MaterialType} and {@link DmgType}. If no sound is defined,
+     * null is returned.
```

Comment:
Maybe return an Optional instead then for these two methods?
UPDATE: In light of later review comments, you may not even want this method at all